File: Java/1544.java
class Solution {
    public String makeGood(String s) {
        int flag = 1,i;
        while(flag == 1){
            flag = 0;
            i = 0;
            while(i<s.length()-1){
                if(s.charAt(i)-s.charAt(i+1) == 32 || s.charAt(i+1)-s.charAt(i) == 32){
                    s = s.substring(0,i) + s.substring(i+2);
                    flag = 1;
                }
                else i++;
            }
        }
        return s;
    }
}

File: Python/1544.py
class Solution(object):
    def makeGood(self, s):
        flag = 1
        while flag:
            flag = 0
            i = 0
            while i < len(s)-1:
                if abs(ord(s[i])-ord(s[i+1])) == 32:
                    s = s[:i] + s[i+2:]
                    flag = 1
                else:
                    i += 1
        return s